src/apps/menu/views/Listings/styled.js

```diff
@@ -49,3 +49,12 @@ export const GridContainer = styled.div`
 export const Flexible = styled.div`
    display: flex;
 `
+
+export const ResponsiveFlex = styled(Flex)`
+   @media screen and (max-width: 767px) {
+      padding: 0px 10px;
+   }
+   @media screen and (min-width: 768px) {
+      padding: 0px 40px;
+   }
+`
```

Comment:
```suggestion
export const ResponsiveFlex = styled(Flex)`
   @media screen and (max-width: 767px) {
      width: calc(100vw - 64px);
   }
   @media screen and (min-width: 768px) {
      width: calc(100vw - 32px);
   }
`
```

instead of removing the `width: calc(100vw - 64px)`, you should've done what's suggested above. Also make sure the spacings are always multiple of 4px in case of margins and paddings.

We use an 4 point grid system. That is any pixel value that is used must be a multiple of 4 except the font-sizes 14, ,18px can still be used.